src/seaice/SeaIce.C

```diff
@@ -1,4 +1,5 @@
 #include "my_f2c.H"
+#include "my_f2c.H"
```

Comment:
Why is this included twice?

Reply:
Never mind this is removed in a later commit

Reply:
Yeah, that snuck in during rebasing at some point, was easier to remove in the last patch, rather than edit the history again